<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>POS Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>365</width>
        <height>470</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>154</red>
            <green>154</green>
            <blue>154</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>167</red>
            <green>167</green>
            <blue>167</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>154</red>
            <green>154</green>
            <blue>154</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>167</red>
            <green>167</green>
            <blue>167</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>167</red>
            <green>167</green>
            <blue>167</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>167</red>
            <green>167</green>
            <blue>167</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QPushButton" name="PayButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>440</y>
         <width>75</width>
         <height>20</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Pay</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BarcodeInput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Barcode</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddToReceipt">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QListWidget" name="Receipt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>261</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="CustomerLookup">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Customer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddCustomer">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QListWidget" name="Customer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="Total">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ModifyCustomer">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
